packages/dashboard/src/features/dashboard/components/observability/ObservabilitySection.tsx

```
@@ -186,49 +192,81 @@ export function ObservabilitySection() {
           timestamp: point.timestamp,
-          value: point.value + (breakdown.wal[i]?.value ?? 0),
+          value: point.value + (densified.wal[i]?.value ?? 0) + (densified.system[i]?.value ?? 0),
         }))
       : diskUsedData;
     return {
-      data: dataUsed,
-      fixedDomain: (dataCeiling !== null ? [0, dataCeiling] : undefined) as
+      data: usedSeries,
+      fixedDomain: (totalBytes !== null ? [axisFloor, totalBytes] : undefined) as
         | [number, number]
         | undefined,
```

ESLint

[error] Replace `|·[number,·number]⏎·······` with `[number,·number]`

(prettier/prettier)